server/src/main/java/io/druid/curator/CuratorConfig.java

```diff
@@ -66,4 +69,14 @@ public void setEnableCompression(Boolean enableCompression)
   {
     this.enableCompression = enableCompression;
   }
+
+  public boolean isEnableAcl()
+  {
+    return enableAcl;
+  }
+
+  public void setEnableAcl(Boolean enableAcl)
```

Comment:
allright that's reasonable. Can it be `Preconditions.notNull(enableAcl, "enableAcl")` then?